File: Python/E-Labsheet/lab11/1.py
n = int(input(""))
j = 1
for i in range (n,0,-1) :
    print("|",end="")
    print((" "*(i-1)),end= "")
    print("*"*j,end= "")
    print((" "*(i-1)),end= "")
    print("|",end="")
    j += 2
    print()

    

File: C/Lab/Lab3/1.c
#include <stdio.h>
#include <stdlib.h>

int main () {
    int n , i = 0 ;
    char input[10];

    fgets(input, sizeof(input), stdin);
    n = atoi(input);
    while (i <= n)
    {
        printf("%d\n",n);
        n--;

    }
    return 0;
}